tests/requirements.txt

```diff
@@ -1 +1,2 @@
-transformers>=4.18
\ No newline at end of file
+transformers>=4.18
```

Comment:
Transformers broke the `PreTrainedModel` class for Tensorizer's usecase from 4.23 onwards until it was fixed in 4.27, maybe this should be set the version to 4.27.

Reply:
Fixed! I had brought it down a while back because 4.18 is the last transformers version compatible with Python 3.6, which we were nominally supporting at the time.